library/src/main/java/com/novoda/downloadmanager/HttpClient.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.downloadmanager;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+interface HttpClient {
```

Comment:
Could have interfaced the `DownloadManagerRequest` too but it seemed overkill and unnecessary.